applications/nrf5340_audio/src/bluetooth/le_audio_bis_headset.c

```diff
@@ -85,6 +85,18 @@ static bool bitrate_check(const struct bt_codec *codec)
 	return true;
 }
 
+static bool adv_data_parse(struct bt_data *data, void *user_data)
```

Comment:
user_data should be const. Do you need the `name` tmp variable?

Reply:
`memcpy((char*)user_data, data->data, data->data_len);` should have the same result.

Reply:
Hmm why should user_data be const? This leads to compile warnings.

![image](https://user-images.githubusercontent.com/89020256/187634265-c2f63ff8-182e-4115-a131-768a42cb140f.png)

Reply:
I included the name variable to follow the same structure as the Zephyr sample code. But I agree that it is not needed for now. Removing it